docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -10,7 +10,7 @@ Here are some typical use cases for running a packaged Kedro project as a Databr
 
 - **Data engineering pipeline**: the output of your Kedro project is a file or set of files containing cleaned and processed data.
 - **Machine learning with MLflow**: your Kedro project runs an ML model; metrics about your experiments are tracked in MLflow.
-- **Automated and scheduled runs**: your Kedro project should be [run on Databricks automatically](https://docs.databricks.com/workflows/jobs/schedule-jobs.html#add-a-job-schedule).
+- **Automated and scheduled runs**: your Kedro project should be [run on Databricks automatically](https://docs.databricks.com/en/jobs/scheduled.html#add-a-job-schedule).
```

Comment:
Alternatively, could link to https://docs.databricks.com/en/jobs/triggers.html#job-schedules--triggers, if want to capture "automated" runs (e.g. file trigger)